### PR TITLE
refactor/remove unnecessary syncserverrequest impl

### PR DESCRIPTION
**Pull request by taddes**
## Description

This task entailed removing unnecessary impl blocks for syncserverrequest, as they were not used. The implementation of the TryFrom trait for HawkIdentifier followed by moving it into `extractors.rs` sufficiently tidied up the code in `mod.rs` scoping the functionality to where the structure is defined and various traits implemented for `HawkIdentifier`

In the Makefile, added a quoting to avoid certain paths with space characters breaking the build. @ethowitz was able to clear this issue up and had dealt with it in the past.

## Testing

Ensuring compilation should be sufficient. Ran into some formatting deltas between CircleCI and clippy in initial tests.

## Issue(s)

Closes [link](link).
https://mozilla-hub.atlassian.net/browse/CONSVC-1693

### CODE REVIEW

File: src/web/extractors.rs [hunk 1]
```diff
@@ -1091,6 +1092,26 @@ impl HawkIdentifier {
...
 
+impl TryFrom<&HttpRequest> for HawkIdentifier {
+    type Error = Error;
+
+    fn try_from(req: &HttpRequest) -> Result<HawkIdentifier, Error> {
+        if DOCKER_FLOW_ENDPOINTS.contains(&req.uri().path().to_lowercase().as_str()) {
+            return Ok(HawkIdentifier::cmd_dummy());
+        }
+        let method = req.method().clone();
+        // NOTE: `connection_info()` gets a mutable reference lock on `extensions()`, so
```
Comment (jrconlin) [inline src/web/extractors.rs:1103]: Looking at what goes on here, the sequence is all blocking code. Is the concern that we don't want the risk of something modifying the `extensions`?

**Review by ethowitz — CHANGES_REQUESTED** [review]
Looks great, @taddes! Just a couple of suggestions 👨🏻‍💻

**Comment by ethowitz** [comment]
I think this needs another rebase after the merging of #1281. Happy to assist if it would be helpful :)

**Review by ethowitz — APPROVED** [review]
Looks great @taddes! :shipit:

**Pull request merged**